### PR TITLE
OFS-285: changes regarding bulk action for FS entities

### PR DESCRIPTION
**Pull request by sniedzielski**
TICKET: https://openimis.atlassian.net/browse/OFS-285

### CODE REVIEW

File: core/gql/gql_mutations/mutation_by_filter.py [hunk 1]
```diff
@@ -107,3 +107,46 @@ def _build_filters_from_gql_filters(filter_fields):
...
+                q_filter = map_gql_to_django_filter(args, available_filters, explicit_filters_handlers)
+
+                from core import datetime
+                now = datetime.datetime.now()
+
+                base_query = django_object \
+                    .objects \
+                    .filter(
+                        Q(date_valid_from=None) | Q(date_valid_from__lte=now),
+                        Q(date_valid_to=None) | Q(date_valid_to__gte=now)
```
Comment (delcroip) [inline core/gql/gql_mutations/mutation_by_filter.py:141]: Q(date_valid_from=None) | Q(date_valid_from__lte=now),

change it to 

Q(date_valid_from__lte=now),

File: core/gql/gql_mutations/mutation_by_filter.py [hunk 2]
```diff
@@ -107,3 +107,46 @@ def _build_filters_from_gql_filters(filter_fields):
...
+
+
+def mutation_on_uuids_from_filter_fs(django_object: django.db.models.Model,
+                                  object_gql_type: DjangoObjectType,
+                                  query_filters_field: str = 'extended_filters',
+                                  explicit_filters_handlers: Dict[str, str] = None,
+                                  return_objects: bool = False):
+    """
+    dedicated extended mutaion from flter decorator dedicated for Formal Sector entities. See doc string for
+    mutation_on_uuids_from_filter to read more how this works.
```
Comment (delcroip) [inline core/gql/gql_mutations/mutation_by_filter.py:119]: please mention GistoryModel BusinessHistoryModel instead for FS (I hope that those model will be used in the future by other)

File: core/gql/gql_mutations/__init__.py [hunk 3]
```diff
@@ -1,3 +1,3 @@
 from .input_types import *
 from .exceptions import ObjectNotExistException, MutationValidationException
-from .mutation_by_filter import mutation_on_uuids_from_filter
+from .mutation_by_filter import mutation_on_uuids_from_filter, mutation_on_uuids_from_filter_fs
```
Comment (delcroip) [inline core/gql/gql_mutations/__init__.py:3]: remane it _history iso _fs

**Review by delcroip — APPROVED** [review]

File: core/gql/gql_mutations/mutation_by_filter.py (hunk 2 above)
Comment (sniedzielski) [reply core/gql/gql_mutations/mutation_by_filter.py:119]: ok, I can rename also this decorator to not use "fs". I will rename to "history_model"

File: core/gql/gql_mutations/mutation_by_filter.py (hunk 1 above)
Comment (sniedzielski) [reply core/gql/gql_mutations/mutation_by_filter.py:141]: ok

File: core/gql/gql_mutations/__init__.py (hunk 3 above)
Comment (sniedzielski) [reply core/gql/gql_mutations/__init__.py]: ok

**Review by delcroip — APPROVED** [review]

**Pull request merged**